test/e2e/tests/consensus_test.go

```diff
@@ -0,0 +1,55 @@
+package e2e_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	nd "github.com/tendermint/tendermint/test/e2e/maverick/node"
+	e2e "github.com/tendermint/tendermint/test/e2e/pkg"
+	"github.com/tendermint/tendermint/types"
+)
+
+func TestEvidence_DoubleVote(t *testing.T) {
```

Comment:
You should probably use `fetchBlockChain()` here to get the complete blockchain history, and then use `testNode` to check that the chain has evidence of that node's misbehavior (if any) - see e.g. `TestValidator_Propose` for a similar case.

Otherwise, this won't work e.g. if the Maverick node also has block pruning enabled, in which case the block with the evidence may have been deleted by the time this test runs.

Reply:
Can i just use status.SyncInfo.FirstBlockHeight and ... LastBlockHeight to check that the height of the attack was within that range

Reply:
But what if the attack happened at height 5, and the node has `FirstBlockHeight` 8?

Reply:
then we just don't make any test assertions about that nodes block store at that height. I'm assuming you are referring to a node that isn't the maverick one right?

Reply:
But the attack still happened, and we should assert that it did.